frontend/src/components/smart/PvPNavBar.vue

```diff
@@ -11,7 +11,13 @@
       </button>
       <button @click="setTab(2)" :class="tabNumber === 2 && 'active'">
         <img src="../../assets/crownIcon.svg" />
-        <span>Rewards</span>
+        <span>Rewards
+          <div class="rewardsIndicator" v-if="hasRewards">
+            <div>
+              <p>!</p>
```

Comment:
I don't know how it will look like in browser, but I think in this case we can use an icon indicator and not literal "!"